### PR TITLE
Fixed the mobile multiline navbar overflow issue

### PR DESCRIPTION
**Pull request by alekangelov**
Really simple fix, just made the container flex on min-width: 395px, as that was the breakpoint.

### CODE REVIEW

**Comment by alekangelov** [comment]
oof I put in min-width instead of max-width, my bad 😂 . All good and working on the netlify preview now.

File: src/layouts/index.css [hunk 1]
```diff
@@ -205,6 +205,12 @@ footer a {
   ##Screen = B/w 320px to 479px
 */
 
+@media (max-width: 395px) {
```
Comment (haxzie) [inline src/layouts/index.css]: It looks like the issue still exists on screen widths between 390px to 400px. It'd be good to set the break point to 400px? do check it out :smile:

**Comment by alekangelov** [comment]
@haxzie changed the breakpoint from 395px to 400px 💯

**Comment by haxzie** [comment]
@alekangelov Looks Good :sparkles: Thank You :smile:

**Pull request merged**